src/common/constants.ts

```diff
@@ -134,3 +135,10 @@ export const TransactionType = {
   COINBASE: 'coinbase' as Transaction['tx_type'],
   POISON_MICROBLOCK: 'poison_microblock' as Transaction['tx_type'],
 } as const;
+
+export const TransactionStatus = {
+  PENDING: 'pending' as TxStatus,
+  SUCCESS_ANCHOR_BLOCK: 'success_anchor_block' as TxStatus,
+  SUCCESS_MICROBLOCK: 'success_microblock' as TxStatus,
+  FAILED: 'failed' as TxStatus,
+};
```

Comment:
I think you can avoid these duplicate type casts if you type as

```suggestion
export const TransactionStatus: Record<string, TxStatus> = {
  PENDING: 'pending',
  SUCCESS_ANCHOR_BLOCK: 'success_anchor_block',
  SUCCESS_MICROBLOCK: 'success_microblock',
  FAILED: 'failed',
};
```

Reply:
Thx for the suggestion, I wasn't sure how to do this.